sncast_std/src/lib.cairo

```diff
@@ -0,0 +1,41 @@
+use starknet::{testing::cheatcode, ContractAddress};
+
+#[derive(Drop, Clone)]
+struct CallResult {
+    data: Array::<felt252>,
+}
+
+fn call(
```

Comment:
If we did it whith syscall::call_contract it would work with dispatchers out of the box

Reply:
Yes, that is the plan.

Reply:
How exactly would you integrate dispatchers with these scripts? Dispatcher generation requires the existence of contract interfaces.

Reply:
for generation, Alex mentioned some tool being developed (abigen?), and we will probably want to use that. Apart from that, no specific plans at the moment, only vague ideas.

Reply:
yep, it is yet to be researched.